### ISSUE TITLE
--osx sends an iOS device to xcodebuild

### ISSUE THREAD

**Issue by kylef** [supermarin/xclisten#17]
``` bash
$ xclisten --osx
Started xclistening to CCLRequestReplay.xcworkspace
xcodebuild -workspace CCLRequestReplay.xcworkspace -scheme CCLRequestReplay -sdk macosx -destination 'name=iPhone Retina (4-inch 64-bit)' test 2> xcodebuild_error.log | xcpretty -tc
```

**Comment by supermarin** [comment]
@kylef i've probably messed up the flags in the last release.
Need to merge @kattrali's acceptance tests and start testing that part

```
            much   thanks
                       such report
```

**Comment by keith** [comment]
+1

**Comment by supermarin** [comment]
Fixed / released in `0.0.8`. Thanks guys!
